optuna/cli.py

```diff
@@ -35,6 +40,28 @@
 
 _DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
 
+# The following code is quoted from the source code of SQLAlchemy project (MIT License).
+# See LICENSE file for detials.
```

Comment:
`The following code` might be a bit clear. It can include all the line after L44, so we might want to specify the implementation of `_rfc1738_pattern`.

Reply:
Thank you for your review. Does the following change look good?

```suggestion
# `_rfc1738_pattern` is quoted from the source code of SQLAlchemy project (MIT License).
# See LICENSE file for details.
```